packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -717,144 +730,145 @@ const AnalyticalTable = forwardRef<AnalyticalTableDomRef, AnalyticalTablePropTyp
           </TitleBar>
         )}
         {extension && <div ref={extensionRef}>{extension}</div>}
-        <FlexBox
-          className={classNames.tableContainerWithScrollBar}
-          data-component-name="AnalyticalTableContainerWithScrollbar"
-        >
-          {showOverlay && (
-            <>
-              <span id={invalidTableTextId} className={classNames.hiddenA11yText} aria-hidden>
-                {invalidTableA11yText}
-              </span>
-              <div
-                tabIndex={0}
-                aria-labelledby={`${titleBarId} ${invalidTableTextId}`}
-                role="region"
-                data-component-name="AnalyticalTableOverlay"
-                className={classNames.overlay}
-              />
-            </>
-          )}
-          <div
-            aria-labelledby={titleBarId}
-            {...getTableProps()}
-            tabIndex={showOverlay ? -1 : 0}
-            role="grid"
-            aria-rowcount={rows.length}
-            aria-colcount={visibleColumns.length}
-            data-per-page={internalVisibleRowCount}
-            data-component-name="AnalyticalTableContainer"
-            ref={tableRef}
-            className={tableClasses}
+        <BusyIndicator active={loading} data-component-name="AnalyticalTableBusyIndicator">
```

Comment:
should be support a `loadingDelay` prop as well?

Reply:
Yup, but will be done in a different PR.